src/Web/Microsoft.NET.Sdk.Web.Targets/Sdk.DefaultItems.targets

```diff
@@ -150,4 +148,15 @@ Copyright (c) .NET Foundation. All rights reserved.
 
     </When>
   </Choose>
+
+  <Target Name="_WarnIfImplicitAspnetCoreAppPackageReferencesAreUsed"
+    Condition=" '$(SuppressImplicitAspNetCorePackageReferenceWarning)' != 'true' ">
+    <Warning
+      Text="The PackageReference to $(_AspNetCoreAllPackageName) is missing the Version attribute. Setting the default version to $(AspNetCoreAllRuntimeFrameworkVersion). See https://aka.ms/aspnetcore/2.1/metapackage-refs for more details about this warning."
+      Condition="'@(_AspNetCoreAllReference->Count())' == '1' AND '@(_ExplicitAspNetCoreAllReference->Count())' == '0'" />
+    <Warning
+      Text="The PackageReference to $(_AspNetCoreAppPackageName) is missing the Version attribute. Setting the default version to $(AspNetCoreAppRuntimeFrameworkVersion). See https://aka.ms/aspnetcore/2.1/metapackage-refs for more details about this warning."
```

Comment:
The link https://aka.ms/aspnetcore/2.1/metapackage-refs doesn't currently exist. Is this a placeholder?

Reply:
just made it. Placeholder for now while we write docs...assuming this gets shiproom approval of course.